pkg/apis/core/v1alpha1/service_bindings.go

```diff
@@ -0,0 +1,24 @@
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+type ServiceBinding struct {
```

Comment:
should this go in `build/v1alpha2` ?

Reply:
i moved this to core because the buildpod package was also using it, but i can explore other options